x86_64-w64-mingw32-g++ cannot link simplest 32bit executable, but x86_64-w64-mingw32-gcc is OK

Using crosstool-ng from master I started from defconfig:
CT_EXPERIMENTAL=y
CT_CC_GCC_V_5_4_0=y
CT_CC_LANG_CXX=y
CT_ARCH_x86=y
CT_ARCH_64=y
CT_MULTILIB=y
CT_TARGET_VENDOR="w64"
CT_KERNEL_windows=y
CT_MINGW_DIRECTX=y
CT_MINGW_DDK=y
CT_MINGW_TOOLS=y
CT_THREADS_NATIVE=y
CT_STATIC_TOOLCHAIN=y

Build is OK, then I wrote simple file bitness.c:
#include <stdio.h>
int tab[1-2*(sizeof(void*)*8 != BITS)];
int main(int argc, char* argv[])
{
    int b = BITS;
    printf("Bitness: %d\n", b);
    return 0;
}
Link results:
./x86_64-w64-mingw32-gcc bitness.c -m64 -DBITS=64  -o w64.exe
./x86_64-w64-mingw32-gcc bitness.c -m32 -DBITS=32  -o w32.exe
./x86_64-w64-mingw32-g++ bitness.c -m64 -DBITS=64  -o w64xx.exe
./x86_64-w64-mingw32-g++ bitness.c -m32 -DBITS=32  -o w32xx.exe
/home/vet/x-tools/x86_64-w64-mingw32/bin/../lib/gcc/x86_64-w64-mingw32/5.4.0/../../../../x86_64-w64-mingw32/bin/ld: cannot find -lstdc++
collect2: error: ld returned 1 exit status

@stilor Alexey,
Could please point me out what is wrong with my default config ?

--- a/scripts/build/cc/100-gcc.sh
+++ b/scripts/build/cc/100-gcc.sh
@@ -759,6 +759,7 @@ do_gcc_for_build() {
 gcc_movelibs() {
     local multi_flags multi_dir multi_os_dir multi_os_dir_gcc multi_root multi_index multi_count
     local gcc_dir dst_dir
+    local rel
 
     for arg in "$@"; do
         eval "${arg// /\\ }"
@@ -779,6 +780,7 @@ gcc_movelibs() {
         dst_dir="${multi_root}/lib/${multi_os_dir}"
     fi
     CT_SanitizeVarDir dst_dir gcc_dir
+    rel=$( echo "${gcc_dir#${CT_PREFIX_DIR}/}" | sed 's#[^/]\{1,\}#..#g' )
 
     ls "${gcc_dir}" | while read f; do
         case "${f}" in
@@ -791,6 +793,7 @@ gcc_movelibs() {
         if [ -f "${gcc_dir}/${f}" ]; then
             CT_DoExecLog ALL mkdir -p "${dst_dir}"
             CT_DoExecLog ALL mv "${gcc_dir}/${f}" "${dst_dir}/${f}"
+            CT_DoExecLog ALL ln -sf "${rel}/${dst_dir#${CT_PREFIX_DIR}/}/${f}" "${gcc_dir}/${f}"
         fi
     done
 }

